www/js/lib/uiUtil.js

```diff
@@ -406,9 +406,9 @@ define(rqDef, function(settingsStore) {
      * Removes the animation effect between various sections
      */
     function removeAnimationClasses() {
-        $('#about').removeClass('slideIn_L').removeClass('slideOut_R');
-        $('#configuration').removeClass('slideIn_L').removeClass('slideIn_R').removeClass('slideOut_L').removeClass('slideOut_R');
-        $('#articleContent').removeClass('slideIn_R').removeClass('slideOut_L');
+        document.getElementById('about').classList.remove('slideIn_L', 'slideOut_R');
+        document.getElementById('configuration').classList.remove('slideIn_L', 'slideIn_R', 'slideOut_L', 'slideOut_R');
+        document.getElementById('articleContent').classList.remove('slideIn_R', 'slideOut_L');
```

Comment:
@Jaifroid , I believe I've figured out the issue.

![image](https://user-images.githubusercontent.com/52416311/204090359-140ebc03-60d6-4c9e-a05d-904613f7624d.png)

The `.remove()` function I've used with multiple parameters is not supported in IE 11 :(
So, I'm pushing a fix.